### ISSUE TITLE
Fails to compile, conflicting types

### ISSUE THREAD

**Issue by E5ten** [andrewgregory/pacutils#30]
```
pacutils/ui.c:34:6: error: conflicting types for 'pu_ui_vwarn'
void pu_ui_vwarn(const char *fmt, va_list args)
     ^
pacutils/../pacutils/ui.h:35:6: note: previous declaration is here
void pu_ui_vwarn(const char *fmt, ...);
     ^
pacutils/ui.c:50:6: error: conflicting types for 'pu_ui_verror'
void pu_ui_verror(const char *fmt, va_list args)
     ^
pacutils/../pacutils/ui.h:34:6: note: previous declaration is here
void pu_ui_verror(const char *fmt, ...);
     ^
pacutils/ui.c:66:6: error: conflicting types for 'pu_ui_vnotice'
void pu_ui_vnotice(const char *fmt, va_list args)
     ^
pacutils/../pacutils/ui.h:36:6: note: previous declaration is here
void pu_ui_vnotice(const char *fmt, ...);
     ^
pacutils/ui.c:223:6: error: conflicting types for 'pu_ui_select_index'
long pu_ui_select_index(long def, long min, long max, const char *prompt, ...)
     ^
pacutils/../pacutils/ui.h:39:5: note: previous declaration is here
int pu_ui_select_index(long def, long min, long max, const char *prompt, ...);
    ^
4 errors generated.
make[1]: *** [Makefile:67: libpacutils.so] Error 1
make: *** [Makefile:4: lib] Error 2
```

**Comment by E5ten** [comment]
I didn't notice that there was an open PR with the fix #27

**Comment by andrewgregory** [comment]
Fixed by b306d91d5d11d00d2895f3ff7d399e5c80915755
